spanner/spanner_samples.rb

```diff
@@ -1873,6 +1873,28 @@ def set_custom_timeout_and_retry project_id:, instance_id:, database_id:
   # [END spanner_set_custom_timeout_and_retry]
 end
 
+def commit_stats project_id:, instance_id:, database_id:
+  # [START spanner_get_commit_stats]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  require "google/cloud/spanner"
+
+  spanner = Google::Cloud::Spanner.new project: project_id
+  client  = spanner.client instance_id, database_id
+
+  records = [
+    { SingerId: 1, FirstName: "Marc",     LastName: "Richards" },
+    { SingerId: 2, FirstName: "Catalina", LastName: "Smith"    }
+  ]
+  resp = client.upsert "Singers", records, commit_stats: true
+  puts "Updated data with #{resp.stats.mutation_count} mutations."
+  puts "Commit was delayed by #{resp.stats.overload_delay} due to overloaded servers."
```

Comment:
It might be better to print this only if it's specified. Otherwise, I believe it's null.

Also specify a time unit (I need to do the same in Node.js).

Reply:
@skuruppu updated.